tests/ChannelHistoryTest.php

```diff
@@ -215,9 +215,14 @@ public function testPublishEventsGetLimitedHistoryAndCheckOrderBackwards() {
 
         $this->assertTrue( $messages2->isPaginated(), 'Expected messages to be paginated' );
         $this->assertTrue( $messages2->hasFirst(), 'Expected to have first page' );
-        $this->assertFalse( $messages2->hasNext(), 'Expected not to have next page' );
-        $this->assertTrue( $messages2->isLast(), 'Expected to be the last page' );
-        $this->assertNull( $messages2->next(), 'Expected the 3rd page to be null' );
+        if ($messages2->hasNext()) {
+            $messages3 = $messages2->next();
+            $this->assertTrue( $messages3->isLast(), 'Expected to be the last page' );
+            $this->assertNull( $messages3->next(), 'Expected the 3rd page to be null' );
```

Comment:
very small typo in the message, should be ``4th`` instead of ``3rd``

Reply:
@jdavid good spot. See https://github.com/ably/ably-php/pull/53/commits/1967ea32deb5279edae98e260d330baeb1183bf1